docs/Calendar/README.md

```diff
@@ -4,9 +4,15 @@
 
 Backpack Android is available through [Jitpack](https://jitpack.io/#Skyscanner/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
 
+Additionally, if you want to handle the output of the calendar (the dates), you also need to rely on the Android adaptation of the JSR-310 backport: [ThreeTenABP](https://github.com/JakeWharton/ThreeTenABP). This is, because we decided to rely on `org.threetenbp.bp.*` constructs as our business entities inside the calendar.
```

Comment:
I suggest we remove this and don't make the lib a `compileOnly` dependency and mention on the usage part how to initialise the lib. Doesn't make much sense to me not to include a lib we expose as part of the `BpkCalendar` public api.

I would also add the code a snippet with `AndroidThreeTen.init(context)` there to showing how to it (plus the link to the full doc)

Reply:
Updated the documentation accordingly